### PR TITLE
Add constructor for the SelectListItem class

### PR DESCRIPTION
**Pull request by FabienLavocat**
I have added a constructor to the `Microsoft.AspNetCore.Mvc.Rendering.SelectListItem` class to set the properties `text`, `value`, `selected` and `disabled` quickly. I've also added a default constructor.

### CODE REVIEW

**Comment by FabienLavocat** [comment]
Thanks for the feedback, I've made the changes.

**Comment by mkArtakMSFT** [comment]
@FabienLavocat, it seems there are some build issues. Please address those before we can take this in.

File: src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultDisplayTemplates.cs [hunk 1]
```diff
@@ -64,24 +64,9 @@ internal static List<SelectListItem> TriStateValues(bool? value)
...
-                    Selected = (value == true),
-                },
-                new SelectListItem
-                {
-                    Text = Resources.Common_TriState_False,
-                    Value = "false",
-                    Selected = (value == false),
-                },
+                new SelectListItem(Resources.Common_TriState_NotSet, string.Empty, !value.HasValue),
+                new SelectListItem(Resources.Common_TriState_True, "true", (value == true)),
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultDisplayTemplates.cs]: I think this 3-param constructor doesn't exist anymore.

Comment (FabienLavocat) [reply src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultDisplayTemplates.cs]: I've made the changes requested, the build works now.

**Pull request merged**